src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -498,7 +498,1246 @@ The user can access any of the four portals using the default credentials: `admi
+      - name: regcred
+      volumes:
+      - name: db-pv-storage
+        persistentVolumeClaim:
+          claimName: db-pv-claim
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: proactive-scheduler
+  name: proactive-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: proactive-scheduler
+  template:
+    metadata:
+      labels:
+        app: proactive-scheduler
+    spec:
+      containers:
+      - envFrom:
+        - configMapRef:
+            name: env-config-g8cf4cd4d8
+        image: dockerhub.activeeon.com/k8s/proactive-scheduler-external-db:12.0.0
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          failureThreshold: 24
+          httpGet:
+            path: /studio
+            port: 8080
+          initialDelaySeconds: 500
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 3
+        name: proactive-scheduler
+        ports:
+        - containerPort: 8080
+        - containerPort: 33647
+        readinessProbe:
+          httpGet:
+            path: /studio
+            port: 8080
+          initialDelaySeconds: 60
+          periodSeconds: 5
+          timeoutSeconds: 3
+        resources:
+          requests:
+            cpu: 1000m
+            memory: 5G
+        volumeMounts:
+        - mountPath: /opt/proactive/server/default
+          name: default-scheduler-pv-storage
+        - mountPath: /opt/proactive/server/previous
+          name: previous-scheduler-pv-storage
+      - command:
+        - sh
+        - -c
+        - /add-node-sources.sh
+        envFrom:
+        - configMapRef:
+            name: env-config-g8cf4cd4d8
+        image: dockerhub.activeeon.com/k8s/proactive-scheduler-external-db:12.0.0
+        imagePullPolicy: IfNotPresent
+        name: proactive-scheduler-nodes
+        stdin: true
+        tty: true
+        volumeMounts:
+        - mountPath: /opt/proactive/server/kube.config
+          name: cluster-config
+          subPath: kube.config
+      imagePullSecrets:
+      - name: regcred
+      initContainers:
+      - command:
+        - sh
+        - -c
+        - until nc -vz -w 3 proactive-database 5432; do echo "Waiting for postgres..."; sleep 3; done;
+        image: busybox:latest
+        imagePullPolicy: IfNotPresent
+        name: wait-for-pg
+      volumes:
+      - name: default-scheduler-pv-storage
+        persistentVolumeClaim:
+          claimName: default-scheduler-pv-claim
+      - name: previous-scheduler-pv-storage
+        persistentVolumeClaim:
+          claimName: previous-scheduler-pv-claim
+      - name: cluster-config
+        secret:
+          secretName: cluster-config-2cd457dbmc
+
+----
+
+
+====== Set up ProActive Pods using MySQL Database
+
+The following configuration could be used to run the ProActive server Pod with an external MySQL Database Pod.
+It will start up to three Pods, one for the server, one for the Nodes, and one for the Database.
```

Comment:
one for the server => one for the Server

Reply:
ok done, I used Server everywhere in the section

Reply:
Yes exactly. This is the reason of my comment 👍🏽